UnrealEngine.gitignore

```diff
@@ -4,6 +4,13 @@
 # Visual Studio 2015 database file
 *.VC.db
 
+# Qt Creator
```

Comment:
These rules are already covered in `QT.gitignore`. Please add any necessary changes to that template so we can avoid duplication of rules.